### ISSUE TITLE
Workflow argument with auto clipboard paste

### ISSUE THREAD

**Issue by liyzcj** [warpdotdev/Warp#3528]
### Describe the solution you'd like?

```Text
I need an argument type, which can automatically paste the content from clipboard to the argument position.
```

### How important is this feature to you?

5 (Can't work without it!)

### Warp Internal (ignore) - linear-label:770f6576-d6c0-4e4f-a259-fc64b5156087

None

**Comment by zheng** [comment]
Just so I understand, is it that you want copying/pasting workflow arguments to preserve the formatting and functionality of workflows?

**Comment by liyzcj** [comment]
yes, when i select the workflow, i want the arguments be pastedautomatically from my clipboard.
